modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java

```diff
@@ -5,6 +5,9 @@
 
 package com.liferay.analytics.reports.web.internal.product.navigation.control.menu;
 
+import com.google.common.cache.Cache;
```

Comment:
@marcosapmf Also, please avoid using Google Cache even if you can find a way to properly manage the result invalidation. Please use portal cache instead. With portal cache (based on Ehcache) we can easily review the hit/miss/other statistics, while with Google Cache we don't have any of those.